buildSrc/.classpath

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Does eclipse not have support for buildSrc via buildship?

Reply:
I don't typically use buildship in eclipse, so I'm not sure. I can always remove the eclipse artifacts.